exercises/dnd-character/lib/dnd_character.ex

```diff
@@ -0,0 +1,15 @@
+defmodule DndCharacter do
+  @spec modifier(pos_integer) :: integer
+  def modifier(score) do
+    # code here
+  end
+
+  @spec ability() :: pos_integer
+  def ability() do
+    # code here
+  end
+
+  def character() do
```

Comment:
A typespec here would be nice. There should be some examples of declaring a type (e.g., defining `.t()`) for the module and then referencing it in functions in this repo.

Reply:
Does defining a `DndCharacter.t()` as a map with required fields make sense?

```elixir
@type t :: %{                     
        strength: pos_integer,    
        dexterity: pos_integer,   
        constitution: pos_integer,
        intelligence: pos_integer,
        wisdom: pos_integer,      
        charisma: pos_integer,    
        hitpoint: pos_integer     
      }                           
```

Reply:
It would be slightly better to use:
```
@type t :: %__MODULE__{                     
        strength: pos_integer,    
        dexterity: pos_integer,   
        constitution: pos_integer,
        intelligence: pos_integer,
        wisdom: pos_integer,      
        charisma: pos_integer,    
        hitpoint: pos_integer     
      }
```

As this communicates it will return the struct of the current module.